test/e2e/pod_stats_test.go

```diff
@@ -177,7 +177,7 @@ var _ = Describe("Podman pod stats", func() {
 
 	It("podman stats on net=host post", func() {
 		// --net=host not supported for rootless pods at present
-		SkipIfRootlessCgroupsV1("Pause stats not supported in cgroups v1")
+		SkipIfRootless("Pause stats not supported in cgroups v1")
```

Comment:
Shouldn't this `Skip` be removed entirely?

Reply:
No, because, It will not work rootless, I believe. It fails locally.  Before the change above it was never being executed in cgroupV2 mode.

The issue is attempting to join the network namespace is causing a permission denied.

Reply:
Oh - got it, I misread the change in `BeforeEach()`. Sorry.

Can you change the string description here, replacing "cgroups v1" with "rootless"?

Reply:
Just grabbed the comment line from above.